### DISCUSSION TITLE
Libdwarf version 0.3.2

### DISCUSSION THREAD

**Discussion by davea42** [davea42/libdwarf-code#49] [Announcements]
Fixes a bug in LEB value reading of some values in the very rare case the compiler has emitted padding in the LEB.  Moved and renamed many files so the directories look cleaner.

<hr /><em>This discussion was created from the release <a href='https://github.com/davea42/libdwarf-code/releases/tag/libdwarf-0.3.2'>Libdwarf version 0.3.2</a>.</em>

**Comment by davea42** [comment]
In two places in src/bin/dwarfdump/print_frames.c there is the code line "offset = offset;"
and with warnings flagged as errors (which --enable-wall does)
and a recent clang as compiler
this stops the build. Which means I need to make a new release. Soonish.

**Reply by alexezeder** [reply]
Maybe this can be checked using GitHub Actions? As well as other tests.
